lambdas/ssm-parameters/tests/ssm-parameters-handler.test.ts

```diff
@@ -0,0 +1,104 @@
+import { SsmParametersHandler } from "../src/ssm-parameters-handler";
+import { Context } from "aws-lambda";
+import { jest } from "@jest/globals";
+import { SSMProvider } from "@aws-lambda-powertools/parameters/ssm";
+
+describe("ssm-parameters-handler", () => {
+  const ssmParametersHandler = new SsmParametersHandler();
+  const ssmProvider = jest.mocked(SSMProvider).prototype;
+  jest.spyOn(ssmProvider, "getParametersByName");
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.resetAllMocks();
+  });
+
+  it("should return list with 1 object when given list of 1 ssmName", async () => {
```

Comment:
Can you change `ssmName` to `SSM name` - doing it in a variable like might incorrectly give context